Speech dictionaries dialog. Added a context menu to the dicts list (#20424)
nvaccess/nvda: user_docs/en/userGuide.md

All dictionary dialogs contain a list of rules which will be used for processing the speech.
The dialog also contains Add, Edit, Remove and Remove all buttons.

> To add a new rule to the dictionary, press the Add button, and fill in the fields in the dialog box that appears and then press Ok.

Reviewer: this line is a duplicate of the first line of the paragraph below?

You will then see your new rule in the list of rules.
All dictionary dialogs contain a list of rules which will be used for processing the speech.

> The dialog also contains Add, Edit, Remove and Remove all buttons.

Reviewer: this is a duplicate of the paragraph above, why are these 2 lines added twice?

To add a new rule to the dictionary, press the Add button, and fill in the fields in the dialog box that appears and then press Ok.
You will then see your new rule in the list of rules.

> As well as the edit and remove buttons, you can also manage individual rules by opening a context menu on an entry in the list (press the Applications key on your keyboard, shift+f10, space bar, or right/left-click), which provides options to Edit or Remove the selected rule.

Reviewer 2: The sentence related to context menu is splitting up two sentences that need to follow each other. Moreover we do not need so many details. Let's just mention that context menu is available and move up the sentence. We do not need to explain here what a context menu is nor how we can open it.

In any case, it's not true that context menu is available through left click: for now it is through left double click, but as requested, this possibility has to be removed anyway.
```suggestion
All dictionary dialogs contain a list of rules which will be used for processing the speech.
The dialog also contains Add, Edit, Remove and Remove all buttons.
Alternatively, a context menu to perform actions on single rules (Edit and Remove) is also available.

To add a new rule to the dictionary, press the Add button, and fill in the fields in the dialog box that appears and then press Ok.
You will then see your new rule in the list of rules.
```

To add a new rule to the dictionary, press the Add button, and fill in the fields in the dialog box that appears and then press Ok.
You will then see your new rule in the list of rules.

> Alternatively, a context menu to perform actions on single rules (Edit and Remove) is also available.

Reviewer: ```suggestion

You can use a context menu to edit or remove rules.
```

> You can also use delete key to remove a selected rule

Reviewer: ```suggestion
You can also use the `delete` key to remove a selected rule
```

To add a new rule to the dictionary, press the Add button, and fill in the fields in the dialog box that appears and then press Ok.
You will then see your new rule in the list of rules.

You can use a context menu to edit or remove rules.

> You can also use the `delete` key to remove a selected rule

Reviewer: ```suggestion
You can also use the `delete` key to remove a selected rule.
```
